ui/src/app/views/research-purpose.tsx

```diff
@@ -0,0 +1,153 @@
+    render() {
+      const {workspace, profileState: {profile}} = this.props;
+      const {publishing, workspacePermissions} = this.state;
+      const selectedResearchPurposeItems = this.getSelectedResearchPurposeItems();
+      const rpItemsHalfLen = sliceByHalfLength(selectedResearchPurposeItems);
+      return <FadeBox style={{margin: '1rem', width: '98%'}}>
+        <div style={styles.mainHeader}>Research Purpose
+          <Clickable disabled={!workspacePermissions.canWrite}
+                     data-test-id='edit-workspace'
+                     onClick={() => navigate(
+                       ['workspaces',  workspace.namespace, workspace.id, 'edit'])}>
+            <EditComponentReact enableHoverEffect={true}
+                                disabled={!workspacePermissions.canWrite}
+                                style={{marginTop: '0.1rem'}}/>
+          </Clickable>
+        </div>
+        <div style={styles.sectionText}>{ResearchPurposeDescription}</div>
+        <div style={styles.sectionHeader}>Primary purpose of the project</div>
+        <TwoColPaddedTable header={false} style={{marginTop: '0.3rem'}}
+                           contentLeft={selectedResearchPurposeItems.slice(0, rpItemsHalfLen)}
+                           contentRight={selectedResearchPurposeItems.slice(rpItemsHalfLen)}/>
+        <div style={styles.sectionHeader}>
+          Reason for choosing <i>All of Us</i> data for your investigation</div>
+        <div style={styles.sectionText}>{workspace.researchPurpose.reasonForAllOfUs}</div>
+        <div style={styles.sectionHeader}>Area of intended study</div>
+        <div style={styles.sectionText}>{workspace.researchPurpose.intendedStudy}</div>
+        <div style={styles.sectionHeader}>Anticipated findings from this study</div>
+        <div style={styles.sectionText}>{workspace.researchPurpose.anticipatedFindings}</div>
+        {workspace.researchPurpose.population && <div style={{marginBottom: '1rem'}}>
+          <div style={styles.sectionHeader}>Population area(s) of focus</div>
+          <TwoColPaddedTable header={false} style={{marginTop: '0.3rem'}}
+                             contentLeft={this.getSelectedPopulationsSlice(true)}
+                             contentRight={this.getSelectedPopulationsSlice(false)}/>
+        </div>
+        }
+        {profile.authorities.includes(Authority.FEATUREDWORKSPACEADMIN) &&
+          <div style={{display: 'flex', justifyContent: 'flex-end'}}>
+              <Button disabled={publishing} type='secondary'
+                onClick={() => this.publishUnpublishWorkspace(false)}>Unpublish</Button>
+              <Button onClick={() => this.publishUnpublishWorkspace(true)}
+                disabled={publishing} style={{marginLeft: '0.5rem'}}>Publish</Button>
+          </div>}
+      </FadeBox>;
+    }
+  }
+);
+
+@Component({
+  selector: 'app-workspace-about',
+  template: '<div #root></div>'
+})
+export class WorkspaceAboutComponent extends ReactWrapperBase {
```

Comment:
Should this be here? Do we need this? If not, we should remove this? If we do need it, we should update the selector and class name.

Reply:
we don't; I will get rid of it.  missed that during my naming update.